features/constants.py

```diff
@@ -7,8 +7,6 @@
 SELENIUM_URL = os.getenv("SELENIUM_URL", "http://127.0.0.1:4444")
 BUGZILLA_API_KEY = os.getenv("BUGZILLA_API_KEY")
 JIRA_API_KEY = os.getenv("JIRA_API_KEY")
-AFFECTED_MODULE_BZ = os.getenv("AFFECTED_MODULE_BZ")
-AFFECTED_MODULE_JR = os.getenv("AFFECTED_MODULE_JR")
```

Comment:
Should I remove this variables from the pipeline?

Reply:
Yes. Please. I remember i already asked you remove these variables.